modules/lang-painless/src/main/java/org/elasticsearch/painless/ir/DoWhileLoopNode.java

```diff
@@ -22,7 +22,7 @@
 import org.elasticsearch.painless.Location;
 import org.elasticsearch.painless.phase.IRTreeVisitor;
 
-public class DoWhileLoopNode extends LoopNode {
+public class DoWhileLoopNode extends ConditionNode {
```

Comment:
?

Reply:
Removed LoopNode as it no longer contained any additional information.